src/main/java/org/jabref/gui/preferences/ImportTab.fxml

```diff
@@ -31,4 +31,20 @@
         <TextField fx:id="fileDirPattern" GridPane.columnIndex="1" GridPane.rowIndex="1"
                    prefWidth="300" minWidth="300" maxWidth="300"/>
     </GridPane>
+
+    <Label styleClass="sectionHeader" text="%API keys"/>
+    <GridPane hgap="4.0" vgap="4.0">
+        <columnConstraints>
+            <ColumnConstraints hgrow="SOMETIMES"/>
+            <ColumnConstraints hgrow="SOMETIMES"/>
+        </columnConstraints>
+        <rowConstraints>
+            <RowConstraints vgrow="SOMETIMES"/>
+            <RowConstraints vgrow="SOMETIMES"/>
+        </rowConstraints>
+        <Label text="%Worldcat Fetcher Key" GridPane.rowIndex="1"/>
```

Comment:
I guess `Fetcher Key` can be removed as this should be clear from the section header already.
It would be nice if could add a link to the site where the user can request the API key (or is this dependent on the library? in this case a "ask your library" comment is appropriate)

Reply:
Renamed it to Worldcat

I added a small description label with the link. However the labels are not selectable and I don't really know how to add a clickable hyperlink in this format either...

Reply:
You can use the `Hyperlink` controls for this (instead of a label). Have a look at the about dialog to do this.

Reply:
Coolio, fixed that!